muttlib/utils.py

```diff
@@ -183,7 +183,7 @@ def clean_quotes(s):
     elif ext == 'feather':
         return pd.read_feather(fn)
     elif ext in ['pickle', 'pkl']:
-        return pd.read_pickle(fn)
+        return pd.read_pickle(fn)  # nosec
```

Comment:
For bandit to pass it's either this or removing the feature.

Reply:
maybe add a comment to entrypoint function cautioning about passing unsafe pickle files because it's a security issue; apart from that i can't think of anything else you can do?

Reply:
Added to docstring (if by entrypoint you mean the `df_read_multi` util).

Reply:
and maybe `local_df_cache()`